tests/server/GraphQLServer.integration.test.ts

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable @typescript-eslint/naming-convention */
+import bodyParser from 'body-parser'
 import express, {Express} from 'express'
 import {Server} from 'node:http'
 import {
     GraphQLServer
-} from '../src/'
+} from '../../src'
```

Comment:
May be start using aliases like 

```
    "paths": {
      "@/*": [
        "src/*"
      ],
      "~/*": [
        "./*"
      ]
    },
```

in `compilerOptions` of `tsconfig.json`

Reply:
Change in every test.